wdl-lint/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+#### Added
+
+* Added `RedundantAssignment` rule ([#444](https://github.com/stjude-rust-labs/wdl/pull/444)).
```

Comment:
I prefer the prior name, `RedundantNone`. "Redundant assignment" is less specific (while still accurate). As this rule only pertains to None values, I think we should revert the name change.

Curious to hear if anyone disagrees.